todoman/cli.py

```diff
@@ -326,11 +338,23 @@ def list(ctx, lists, all, urgent, location, category, grep, sort, reverse):
                (not pattern or (
                    pattern.search(todo.summary) or
                    pattern.search(todo.description)
-                   ))
+                )) and
+               (
```

Comment:
On all other lines the following style is used:

    (a
     b)

but here you use:

    (
      a
      b
    )